package.json

```diff
@@ -35,15 +35,19 @@
   },
   "scripts": {
     "webpack": "webpack",
-    "dev": "yarn run webpack -- --env mode=development",
-    "prod": "yarn run webpack -- --env mode=production",
-    "prod:analyze": "yarn run prod -- --env presets=analyze",
+    "dev": "yarn run webpack --env mode=development",
+    "prod": "yarn run webpack --env mode=production",
+    "prod:analyze": "yarn run prod --env presets=analyze",
     "build": "yarn run prod",
     "start": "yarn run dev && http-server -p 3000 dist --allowed-hosts=all",
     "serve:prod": "http-server -p 3000 dist",
     "generateimportmap": "(cd public && jspm link index.html -o index.html)",
     "web4:deploy": "NEAR_ENV=mainnet npx web4-deploy dist psalomobos.near --nearfs",
-    "test": "npx playwright test"
+    "test": "npx playwright test",
+    "test:watch": "npx playwright test --ui",
+    "test:watch:codespaces": "npx playwright test --ui-host=0.0.0.0",
```

Comment:
How should we be using these test:watch commands?

Reply:
Thanks, I've renamed to `test:ui` as that makes more sense, and I've also added more info about this in README.md